### PR TITLE
Deprecated tx history

### PR DESCRIPTION
**Pull request by stackchain**

### CODE REVIEW

File: src/TxHistory/TxDetails.tsx [hunk 1]
```diff
@@ -88,67 +80,58 @@ const TxDetails = () => {
 
       <OfflineBanner />
       <Screen scroll>
-        <Banner label={intl.formatMessage(txTypeMessages[transaction.direction])}>
+        <Banner label={strings[transaction.direction]}>
```
Comment (stackchain) [inline src/TxHistory/TxDetails.tsx:83]: - check if ts will complain if the styles go away

File: src/TxHistory/TxHistoryNavigator.tsx [hunk 2]
```diff
@@ -1,28 +1,45 @@
...
+export const TxHistoryNavigator = () => {
+  const walletMeta = useSelector(walletMetaSelector)
+  const transactionInfos = useSelector(transactionsInfoSelector)
 
-type TxHistoryRoutes = {
-  'tx-history-list': any,
-  'tx-details': any,
-}
+  return (
+    <Stack.Navigator screenOptions={{...defaultStackNavigatorOptions}} initialRouteName={TX_HISTORY_ROUTES.MAIN}>
```
Comment (stackchain) [inline src/TxHistory/TxHistoryNavigator.tsx]: remove it

File: src/WalletNavigator.tsx [hunk 3]
```diff
@@ -25,6 +24,7 @@ import {CatalystNavigator} from './Catalyst/CatalystNavigator'
 import {ReceiveScreenNavigator} from './Receive/ReceiveScreenNavigator'
 import {useSelectedWallet, WalletSelectionScreen} from './SelectedWallet'
 import {SettingsScreenNavigator} from './Settings'
+import {TxHistoryNavigator} from './TxHistory/TxHistoryNavigator'
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
 type WalletTabRoutes = {
```
Comment (stackchain) [inline src/WalletNavigator.tsx:30]: add icons to storybook - imgs

**Review by wolverineks — APPROVED** [review]

File: src/TxHistory/TxHistoryNavigator.tsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+          ...defaultNavigationOptions,
+        })}
+      />
+    </Stack.Navigator>
+  )
+}
+
+const styles = StyleSheet.create({
+  settingsButton: {
+    height: '80%',
```
Comment (wolverineks) [inline src/TxHistory/TxHistoryNavigator.tsx]: in general, avoid percentages
also, i think in this case you can safely remove this style altogether

File: src/TxHistory/WalletHero/BalanceBanner.tsx [hunk 5]
```diff
@@ -19,7 +21,7 @@ const QUOTE_PAIR_CURRENCY = 'USD'
 export const BalanceBanner = () => {
   const intl = useIntl()
   const tokenBalance = useSelector(tokenBalanceSelector)
-  // const walletMeta = useSelector(walletMetaSelector)
+  const walletMeta = useSelector(walletMetaSelector)
```
Comment (wolverineks) [inline src/TxHistory/WalletHero/BalanceBanner.tsx]: when possible, prefer `const wallet = useSelectedWallet()`

**Pull request merged**